src/items/equipment/tool_kit_akinzi_resources_catalyst_rig.json

```diff
@@ -0,0 +1,106 @@
+{
+  "_id": "FwgsbBbZhcMMHTeX",
+  "name": "Tool kit, Akinzi Resources, Catalyst Rig",
+  "type": "technological",
+  "flags": {
+    "core": {
+      "sourceId": "Compendium.sfrpg.equipment.oZXCkSuecrXo2b4L"
+    }
+  },
+  "img": "systems/sfrpg/icons/equipment/technological%20items/tool-kit-starfinder-armory.webp",
+  "system": {
+    "type": "",
+    "abilityMods": {
+      "parts": []
+    },
+    "ammunitionType": "",
+    "attributes": {
+      "ac": {
+        "value": ""
+      },
+      "customBuilt": false,
+      "dex": {
+        "mod": ""
+      },
+      "hardness": {
+        "value": ""
+      },
+      "hp": {
+        "max": "",
+        "value": 6
+      },
+      "size": "medium",
+      "sturdy": false
+    },
+    "attuned": false,
+    "bulk": "L",
+    "capacity": {
+      "max": 0,
+      "value": 0
+    },
+    "container": {
+      "contents": [],
+      "includeContentsInWealthCalculation": true,
+      "isOpen": true,
+      "storage": []
+    },
+    "critical": {
+      "parts": []
+    },
+    "damage": {
+      "parts": []
+    },
+    "description": {
+      "chat": "",
+      "gmnotes": "",
+      "short": "",
+      "unidentified": "",
+      "value": "<p>Akinzi Resources, a startup company operating out of shared office space on Verces, produces a line of specialty toolkits that supplement those presented on page 221 of the Core Rulebook. These toolkits provide circumstance bonuses to one or more skills in specific situations and include the following: tourist’s kit (grants a +4 bonus to Culture checks to recall knowledge about related to a specific planet), <strong>catalyst rig</strong> (increases your effective skill ranks by 2 for the purpose of determining how long it takes you to craft drugs, poisons, or medicinals), influencer kit (grants a +4 bonus to Diplomacy checks made to change attitude for a specific subculture like mercenaries or augmentation enthusiasts), gymnast’s kit (grants a +4 bonus to Acrobatics checks made to tumble), swimmer’s kit (grants a +4 bonus to Athletics checks made to swim), grifter’s kit (grants a +4 bonus to Sleight of Hand checks made to entertain and pick pocket), and tracker’s kit (grants a +4 bonus to Survival checks made to follow tracks).</p>"
```

Comment:
For the description on these, you can remove all the other toolkit descriptions so it's something like

```
Akinzi Resources, a startup company operating out of shared office space on Verces, produces a line of specialty toolkits that supplement those presented on page 221 of the Core Rulebook. These toolkits provide circumstance bonuses to one or more skills in specific situations and include the following:
Catalyst Rig: ...
```

Reply:
I had originally done that and Lankid told me to leave them in.  Will correct